state_manager/state_manager_cli.py

```diff
@@ -0,0 +1,53 @@
+# Util to access the state manager.
+# To send data to the state manager, enter 4 whitespace-separated numbers and hit enter.
```

Comment:
Would it just be easier to have them enter hex numbers instead of ints?
Since you are just converting to a hex value anyways?